### PR TITLE
951-View dataroom folder tree list and documnet list ui fix

### LINKED ISSUE #951: [🕹️] Dataroom view ui : hotFix
### What side quest or challenge are you solving?

Always show scroll and on bottom bg-color

image

if List is big!!!

image

### Points

@mfts ?

### Description

_No response_

### Provide proof that you've completed the task

PR

### PR DESCRIPTION
**Pull request by AshishViradiya153**
fixes #951

### CODE REVIEW

File: components/view/DataroomViewer.tsx [hunk 1]
```diff
@@ -153,89 +151,100 @@ export default function DataroomViewer({
...
+                        </React.Fragment>
+                      ))}
+                    </BreadcrumbList>
+                  </Breadcrumb>
+                </div>
+                <ul role="list" className="space-y-4 overflow-auto p-4">
+                  {mixedItems.map(renderItem)}
+                </ul>
+                <ScrollBar orientation="horizontal" />
+                <ScrollBar orientation="horizontal" />
```
Comment (mfts) [inline components/view/DataroomViewer.tsx]: why are there two scrollbar components?

File: components/view/DataroomViewer.tsx [hunk 2]
```diff
@@ -153,89 +151,100 @@ export default function DataroomViewer({
...
+          <div className="hidden h-full w-1/4 space-y-8 overflow-auto px-3 pb-4 pt-4 md:flex md:px-6 md:pt-6 lg:px-8 lg:pt-9 xl:px-14">
+            <ScrollArea showScrollbar>
+              <ViewFolderTree
+                folders={folders}
+                documents={documents}
+                setFolderId={setFolderId}
+                folderId={folderId}
+              />
+              <ScrollBar orientation="horizontal" />
+              <ScrollBar orientation="horizontal" />
```
Comment (mfts) [inline components/view/DataroomViewer.tsx]: why are there two scrollbar components?

**Review by mfts — CHANGES_REQUESTED** [review]
quick question about some duplicate components

File: components/view/DataroomViewer.tsx (hunk 2 above)
Comment (AshishViradiya153) [reply components/view/DataroomViewer.tsx]: sorry, i made mistake...

**Comment by AshishViradiya153** [comment]
now perfect, @mfts recheck please
Thanks.

**Pull request merged**